lib/ddtrace/contrib/rails/core_extensions.rb

```diff
@@ -9,15 +9,33 @@ def patch_renderer
     end
 
     def patch_renderer_render_template
-      if defined?(::ActionView::Renderer)
-        ::ActionView::Renderer.class_eval do
+      if defined?(::ActionView::TemplateRenderer)
+        ::ActionView::TemplateRenderer.class_eval do
+          alias_method :render_without_datadog, :render
           alias_method :render_template_without_datadog, :render_template
-          def render_template(*args, &block)
-            ActiveSupport::Notifications.instrument('start_render_template.action_view')
-            render_template_without_datadog(*args, &block)
+
+          def render(context, options)
```

Comment:
This is the first called method of a `TemplateRenderer`. We should patch this one so that we can catch any rendering exception (included exceptions that doesn't trigger a rendering).